uml/backend-software-system/input-listener.puml

```diff
@@ -0,0 +1,53 @@
+@startuml input-listener-console
+package "Console" {
+
+    interface IEventListener {
+        +handler: IEventHandler
```

Comment:
interfaces cannot have class attributes.

Reply:
sorry, It's not an interface, i'ts an abstract class because the contract is dependent on its attributes, the same thing happens with IEventHandler.

You can see it in the design th
![Screenshot from 2024-05-31 10-15-58](https://github.com/Pending-Name-21/arquitecture/assets/135136456/a86b7680-c766-41d6-b152-a1c7f0c99fd6)
at was built first, so the interface per abstract class will be corrected:

Reply:
nice, please do update the puml